examples/getting_started/tf/nvflare_tf_getting_started.ipynb

```diff
@@ -397,8 +427,8 @@
    "id": "9ac3f0a8-06bb-4bea-89d3-4a5fc5b76c63",
    "metadata": {},
    "source": [
-    "#### 7. Run FL Simulation\n",
-    "Finally, we can run our FedJob in simulation using NVFlare's [simulator](https://nvflare.readthedocs.io/en/main/user_guide/nvflare_cli/fl_simulator.html) under the hood. The results will be saved in the specified `workdir`."
+    "#### 8. Run FL Simulation\n",
```

Comment:
content-wise LGTM, this example might hang, have you tried it locally?

Reply:
Yes, I've tested all of these examples and they work